### PR TITLE
Release v0.9.11

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/blog/releases/v0.9.11.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+description: >
+  Release v0.9.11
+---
+
+## Changelog since v0.9.10
+
+### Notable Changes
+* Disable pipectl datastore command ([#1843](https://github.com/pipe-cd/pipe/pull/1843))
+* Disable MongoDB as ControlPlane's datastore ([#1835](https://github.com/pipe-cd/pipe/pull/1835))
+
```
Comment (nakabonne) [inline docs/content/en/blog/releases/v0.9.11.md:14]: Could you please add "Piped now works with namespace scope resources as well as cluster scope."?

Comment (nghialv) [reply docs/content/en/blog/releases/v0.9.11.md:14]: Where is the PR for that?

Comment (nakabonne) [reply docs/content/en/blog/releases/v0.9.11.md:14]: Here: https://github.com/pipe-cd/pipe/pull/1830

Comment (nakabonne) [reply docs/content/en/blog/releases/v0.9.11.md:14]: Already exists at L24: `Update piped to not create cluster-level resources ([#1830](https://github.com/pipe-cd/pipe/pull/1830))`

Comment (nghialv) [reply docs/content/en/blog/releases/v0.9.11.md:14]: OK. From the next times, you can fill that description into the PR's body at
`Does this PR introduce a user-facing change?:`

Comment (nakabonne) [reply docs/content/en/blog/releases/v0.9.11.md:15]: Right. I have totally forgotten. I'm going to do so.

**Comment by nakabonne** [comment]
Thank you!
/approve

File: docs/content/en/blog/releases/v0.9.11.md (hunk 1 above)
Comment (nghialv) [reply docs/content/en/blog/releases/v0.9.11.md:15]: 👍  I have just fixed the changelog.

**Pull request merged**